src/features/account/components/trusted-circle/TrustedCircleTabPanel.tsx

```diff
@@ -0,0 +1,6 @@
+import { Box } from '@chakra-ui/react';
+import type { FunctionComponent } from 'react';
+
+export const TrustedCircleTabPanel: FunctionComponent = () => {
```

Comment:
How is this used?

Reply:
it will have more components in the up coming story, for demo purpose , we leave it empty